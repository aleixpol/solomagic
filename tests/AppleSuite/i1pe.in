\ref something0
\mt somethingsomething inami somethingsomething

\ref something1
\mt somethingsomething inanimate somethingsomething

