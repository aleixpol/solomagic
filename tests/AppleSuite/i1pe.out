\ref something0
\mt somethingsomething inami somethingsomething
\ma somethingsomething ingnami somethingsomething

\ref something1
\mt somethingsomething inanimate somethingsomething
\ma somethingsomething inanimate somethingsomething
